README.md

```diff
@@ -5,30 +5,29 @@
 
 Scans domains for data on their HTTPS configuration and assorted other things.
 
-**Most of the work is farmed out to other command line tools.** The point of this project is to **coordinate** those tools and produce **consistent data output**.
+**Much of this work is farmed out to other command line tools.**
 
-Can be used with any domain, or CSV where domains are the first column, such as the [official .gov domain list](https://catalog.data.gov/dataset/gov-domains-api-c9856).
+The point of this project is to **coordinate** and **parallelize** those tools and produce **consistent data output**.
 
-### Requirements
+Can be used with any domain, or any CSV where domains are the first column, such as the [official .gov domain list](https://github.com/GSA/data/raw/gh-pages/dotgov-domains/current-full.csv).
 
-The requirements here can be quite diverse, because this tool is just a coordinator for other tools. Communication between tools is handled via CLI and STDOUT.
+### Requirements
 
-The overall tool requires **Python 3**. To install dependencies:
+`domain-scan` requires **Python 3**. To install dependencies:
 
 ```bash
 pip install -r requirements.txt
 ```
 
-The individual scanners each require their own dependencies. You only need to have the dependencies installed for the scanners you plan to use.
+Some individual scanners require externally installed dependencies:
 
-* `pshtt` scanner: **Python 2** and **[pshtt](https://github.com/dhs-ncats/pshtt)**, ideally installed with `pyenv` via `pip install pshtt`.
-* `tls` scanner: **Go** and **[ssllabs-scan](https://github.com/ssllabs/ssllabs-scan)**, stable branch.
-* `sslyze` scanner: **[sslyze](https://github.com/nabla-c0d3/sslyze)** 1.0 or greater (installed automatically via `requirements.txt`).
-* `pageload` scanner: **Node** and **[phantomas](https://www.npmjs.com/package/phantomas)**, installed through npm.
+* `pshtt` scanner: The `pshtt` command, available from the [`pshtt`](https://github.com/dhs-ncats/pshtt) Python package.
+* `tls` scanner: The `ssllabs-scan` command, available by compiling and installing from the Go-based [ssllabs-scan](https://github.com/ssllabs/ssllabs-scan) repo (stable branch).
+* `pageload` scanner: The `phantomas` command, available from the [`phantomas`](https://www.npmjs.com/package/phantomas) Node package.
```

Comment:
Reworded this section of the README so that it focuses on what's actually needed -- specific CLI commands available, for some scanners (as opposed to random languages and dependencies for most scanners).